db/migrate/20211029174211_drop_source_columns.rb

```diff
@@ -0,0 +1,6 @@
+class DropSourceColumns < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :spree_adjustments, :source_id
+    remove_column :spree_adjustments, :source_type
```

Comment:
I think that this can lead to downtime until the app code is reloaded. Rails can still try to insert adjustments with these columns.

We can include `self.ignored_columns = %w(source_id source_type)` into the model and deploy that first. Then this PR follows in the next release.

I think that it's also a good idea to include the data type to make this reversible:
```suggestion
    remove_column :spree_adjustments, :source_id, :integer
    remove_column :spree_adjustments, :source_type, :string
```

Reply:
> Rails can still try to insert adjustments with these columns.

There shouldn't be any code that's actually using these columns though...?

Reply:
Hm, you are right! I thought that I observed Rails trying to set that column anyway. It probably applies only to NOT NULL columns. But these ones are optional and therefore it's safe. :+1: